.github/workflows/build_test.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Test
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }} # compiles and test on Ubuntu
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java: ["11"]
+      fail-fast: false
+
+    steps:
+    - name: Checkout PR branch
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    # Checkout global JBT settings.xml
+    - name: Checkout JBoss Tools Build CI
+      uses: actions/checkout@v2
+      with:
+        repository: jbosstools/jbosstools-build-ci
+        path: build-ci
+
+    # Java JDK 11 used for maven build
+    - name: Setup Java ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64
+
+    # Try to find and apply jbosstools cache
+    - name: Cache local Maven repository for JBoss Tools components
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: jbosstools-${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          jbosstools-${{ runner.os }}-maven-
+
+    # Build and compile using Maven
+    - name: Build/Compile and run unit tests
+      uses: GabrielBB/xvfb-action@v1
+      with:
+        run: mvn clean verify --settings build-ci/maven-settings.xml --quiet
```

Comment:
--quiet is difficult to review when failing. please use only --no-transfer-progress